packages/suite/src/utils/wallet/cardanoUtils.ts

```diff
@@ -0,0 +1,262 @@
+    accountTokens: Account['tokens'],
+    maxOutputIndex?: number,
+): types.UserOutput[] =>
+    outputs.map((output, i) => {
+        const setMax = i === maxOutputIndex;
+        const amount = output.amount === '' ? undefined : amountToSatoshi(output.amount, 6);
+        const tokenDecimals = accountTokens?.find(t => t.address === output.token)?.decimals ?? 0;
+        return {
+            address: output.address === '' ? undefined : output.address,
+            amount: output.token ? undefined : amount,
+            assets: output.token
+                ? [
+                      {
+                          unit: output.token,
+                          quantity: output.amount
+                              ? amountToSatoshi(output.amount, tokenDecimals)
+                              : '0',
+                      },
+                  ]
+                : [],
+            setMax,
+        };
+    });
+
+export const getShortFingerprint = (fingerprint: string) => {
+    const firstPart = fingerprint.substring(0, 10);
+    const lastPart = fingerprint.substring(fingerprint.length - 10);
+    return `${firstPart}…${lastPart}`;
+};
+
+export const transformUtxos = (utxos: Account['utxo']): types.Utxo[] => {
+    const result: types.Utxo[] = [];
+    utxos?.forEach(utxo => {
+        const foundItem = result.find(
+            res => res.txHash === utxo.txid && res.outputIndex === utxo.vout,
+        );
+
+        if (utxo.cardanoSpecific) {
+            if (!foundItem) {
+                // path: utxo.path,
+                result.push({
+                    // path: utxo.path,
+                    address: utxo.address,
+                    txHash: utxo.txid,
+                    outputIndex: utxo.vout,
+                    amount: [{ quantity: utxo.amount, unit: utxo.cardanoSpecific.unit }],
+                });
+            } else {
+                foundItem.amount.push({ quantity: utxo.amount, unit: utxo.cardanoSpecific.unit });
+            }
+        }
+    });
+
+    return result;
+};
+
+export const prepareCertificates = (certs: CardanoCertificate[]) => {
+    // convert trezor-connect certificate format to cardano coin-selection lib format
+    const convertedCerts: types.Certificate[] = [];
+    certs.forEach(cert => {
+        switch (cert.type) {
+            case CardanoCertificateType.STAKE_DELEGATION:
+                convertedCerts.push({
+                    type: cert.type,
+                    pool: cert.pool!,
+                });
+                break;
+            case CardanoCertificateType.STAKE_REGISTRATION:
+            case CardanoCertificateType.STAKE_DEREGISTRATION:
+                convertedCerts.push({
+                    type: cert.type,
+                });
+                break;
+            // no default
+        }
+    });
+    return convertedCerts;
+};
+
+export const parseAsset = (
+    hex: string,
+): {
+    policyId: string;
+    assetNameInHex: string;
+} => {
+    const policyIdSize = 56;
```

Comment:
could 56 be also candidate for moving into some constants file?

Reply:
I don't expect it will be used anywhere else, ever. Do you still want to move it to constants file?